Lib/test/test_richcmp.py

```diff
@@ -253,6 +253,8 @@ def test_recursion(self):
         self.assertTrue(a != b)
         self.assertTrue(a < b)
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
```

Comment:
I think this should be pretty easy to fix, if you want - I think it's on line 631 in vm.rs, we currently differ from CPython's message format, which is `'{}' not supported between instances of '{}' and '{}'`

Reply:
Resolved in 4c8e402.